### PR TITLE
Add payment methods

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: spec/definitions/PaymentInstrument.yaml [hunk 1]
```diff
@@ -4,10 +4,4 @@ required:
...
 properties:
   method:
-    description: The payment method
-    type: string
-    enum:
-      - ach
-      - cash
-      - payment-card
-      - paypal
+    $ref: "#/definitions/Method"
```
Comment (colinbird) [inline spec/definitions/PaymentInstrument.yaml]: is this ok, or breaks the discriminator ?

File: spec/definitions/Method.yaml [hunk 2]
```diff
@@ -6,4 +6,5 @@ enum:
   - china-unionpay
   - payment-card
   - paypal
-
+  - OchaPay
```
Comment (slavcodev) [inline spec/definitions/Method.yaml]: Just I not sure there we need to use brand name.

Comment (adamaltman) [inline spec/definitions/Method.yaml]: Yes, and I think we use it, as well as `China UnionPay`.

Comment (slavcodev) [inline spec/definitions/Method.yaml]: Usage of brand name as gateway name makes sense for me, but there is instrument type, it's seems our internal things. It's looks confusing to have `payment-card` and `OchaPay` in one enum.

Comment (adamaltman) [inline spec/definitions/Method.yaml]: It seems we made a policy to use brand names with appropriate brand casing where brands are used.  For example, above we have `paypal`, but the brand is actually `PayPal`.

**Pull request merged**